mangaki/mangaki/utils/work_merge.py

```diff
@@ -107,10 +113,21 @@ def field_changeset(works):
         elif sum(is_empty_field(choice) for choice in choices) == len(choices) - 1:
             action = ActionType.JUST_CONFIRM
             suggested = [choice for choice in choices if not is_empty_field(choice)][0]
+        elif field == 'ext_poster' and all(strip_protocol.sub('', choice) == strip_protocol.sub('', choices[0]) for choice in choices):
```

Comment:
While using regular expression power is nice, I suggest we use proper URL parsing using https://docs.python.org/3/library/urllib.parse.html for example.